### ISSUE TITLE
Alternate `g` not working consistently

### ISSUE THREAD

**Issue by rajadain** [eigilnikolajsen/commit-mono#35]
I downloaded Commit Mono (thank you for making this, it's my new favorite font!) with the alternative (single-story) `g`, as so:

![image](https://github.com/eigilnikolajsen/commit-mono/assets/1430060/f87f2da9-2a40-4bb1-9423-a677e29a83b7)

However, it doesn't work consistently. It seems to be triggered via a ligature-like behavior, where it only shows up in certain pairings, rather than replacing the character in the downloaded font, which is seemingly done with the dotted `0`, and the `6` and `9` characters. In some cases (e.g. Sublime Text), it never works. In others (e.g. Chrome), it works unexpectedly, for example:

![image](https://github.com/eigilnikolajsen/commit-mono/assets/1430060/6fe4607f-120e-4b27-8d74-86e03307d64b)

Can the `g` be changed to behave like the `0`, `6`, and `9` characters please? Thank you.

This may be related to #25.

**Comment by eigilnikolajsen** [comment]
That's some odd behaviour. I will fix it asap, thank you for posting!

**Comment by eigilnikolajsen** [comment]
Hello @rajadain, I tried to reproduce the issue but unsuccessfully. Can you provide a bit more detail to how I can reproduce it? Also, can you please upload your font files here, so I can check them?

**Comment by rajadain** [comment]
Here's the files I'm using: [CommitMono-lig-w500-h125-altG60It.zip](https://github.com/eigilnikolajsen/commit-mono/files/12440196/CommitMono-lig-w500-h125-altG60It.zip)

I went with ligatures and arrows on, weight 500, height 1.25, and alts for G, 6/9, dotted 0, and the italics.

I'm just using the fonts in VS Code and iTerm2. In Chrome I also use Stylus to change CSS styles for GitHub, which is basically this:

```css
  pre, code, tt, kbd:not(.badmono), samp, .blob-code, .file-data pre, .line-data,
  #gist-form .file .input textarea, .blob-code-inner, .react-code-text, .jujkut, .input-monospace,
  #read-only-cursor-text-area {
    font-family: "CommitMono-lig-w500-h125-altG60It", Consolas, "Liberation Mono", Menlo, Courier, monospace !important;
    font-feature-settings: normal !important;
    font-variant-ligatures: normal !important;
    font-size: 14px !important;
  }
```

Thank you for looking in to this!

**Comment by eigilnikolajsen** [comment]
This should be fixed in [version 1.135](https://github.com/eigilnikolajsen/commit-mono/releases/tag/1.135)!

**Comment by rajadain** [comment]
Thank you so much for taking care of this!!!
